notebooks/003-hello-segmentation/003-hello-segmentation.ipynb

```diff
@@ -0,0 +1,191 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Hello Segmentation\n",
+    "\n",
+    "A very basic introduction to OpenVINO segmentation model.\n",
+    "\n",
+    "We use a [Semantic Segmentation Adas](https://docs.openvinotoolkit.org/2019_R1/_semantic_segmentation_adas_0001_description_semantic_segmentation_adas_0001.html) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/).\n"
```

Comment:
ADAS is not a very well known term so we could add a sentence explaining what it is https://www.adasandme.com/news/adas-is-now-adaptive-driver-assistance-systems/
and also have a sentence that says that this model recognizes four classes: background, road, curb and mark. 

```suggestion
    "We use the [road-segmentation-adas-0001](https://docs.openvinotoolkit.org/latest/omz_models_model_road_segmentation_adas_0001.html) model from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/).\n"
```

Reply:
When you search for OMZ models sometimes outdated documents are the first in the search results (you see that in the 2019_R1 in your link). We should link to the most recent version.

Reply:
Applied suggestions and added some description. Can you check is this what you ment?